### PR TITLE
PS-2423 switch to token authentication

### PR DESCRIPTION
**Pull request by pivnicek**
As described here: https://keboola.atlassian.net/browse/PS-2423

### CODE REVIEW

File: notebookUtils.py [hunk 1]
```diff
@@ -180,7 +180,7 @@ def notebookSetup(c):
 
     # Set a password
     if 'PASSWORD' in os.environ and os.environ['PASSWORD']:
```
Comment (MiroCillik) [inline notebookUtils.py:182]: The ENV name won't change to TOKEN as well?

**Review by MiroCillik — APPROVED** [review]
That was easy :)

File: notebookUtils.py (hunk 1 above)
Comment (pivnicek) [reply notebookUtils.py:182]: nope, we're going to just use the same one, no real need to change it.

Comment (MiroCillik) [reply notebookUtils.py:182]: Ok

**Comment by pivnicek** [comment]
The trick is making sure that it works in https://github.com/keboola/sandboxes-jupyterlab :-) which is the next step

**Pull request merged**